File: csharp/isogram/Isogram.cs
using System;
using System.Linq;

public static class Isogram
{
    public static bool IsIsogram(string word) => !word.Where(char.IsLetter)
                                                     .GroupBy(char.ToLower)
                                                     .Any(g => g.Count() > 1);
}

File: fsharp/isogram/Isogram.fs
module Isogram

let isIsogram (str: string) =
    str.ToLowerInvariant()
    |> Seq.filter System.Char.IsLetter
    |> Seq.countBy (fun c -> c)
    |> Seq.forall (fun (_, x) -> x = 1)
